### PR TITLE
QR scanner documentation

### PR DESCRIPTION
**Pull request by sanyud24**
closes odk-x/tool-suite-X#267

addresses odk-x/tool-suite-X#

#### What is included in this PR?

Documentation on how to use QR Scanner

### CODE REVIEW

File: odkx-src/services-using.rst [hunk 1]
```diff
@@ -62,6 +62,23 @@ Within this page you can enter a new :guilabel:`username` and :guilabel:`passwor
 
 If you want to log out of your current user without logging into a new user, click the :guilabel:`Log Out` button. This does not require a network connection.
 
+.. _services-using-qr-scanner:
+
+Authenticating Users with QR-scanner
+---------------------------------------
+
+Users can use this option as an alternative to above method by logging in using QR code Scanner, that can be generated by app-designer.
```
Comment (elmps2018) [inline odkx-src/services-using.rst:70]: This sentence was a bit unclear/repetitive to me. Maybe "Users can alternatively log in using the QR code scanner, with QR codes generated by the ODK-X app-designer"

File: odkx-src/services-using.rst [hunk 2]
```diff
@@ -62,6 +62,23 @@ Within this page you can enter a new :guilabel:`username` and :guilabel:`passwor
...
+
+   1. Press the Action Button (:guilabel:`⋮`) and select "Setting" options.
```
Comment (elmps2018) [inline odkx-src/services-using.rst:72]: Is it Setting or Settings?

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 3]
```diff
@@ -98,7 +98,9 @@ Setup instructions:
 
     .. note::
 
-      It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
+      It is important that the right side of the colon stays as 8080 or whatever port you are using (8080 is the default). This is the port that the web server is looking for.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-manual-setup.rst:101]: This part does not seem to be about the QR scanner, does it relate to another open pull request?

**Review by elmps2018 — COMMENTED** [review]
Thanks for working on this issue!

**Comment by sanyud24** [comment]
I'll correct everything

File: odkx-src/sync-endpoint-manual-setup.rst (hunk 3 above)
Comment (sanyud24) [reply odkx-src/sync-endpoint-manual-setup.rst:101]: I'll just delete this part

**Comment by elmps2018** [comment]
@sanyud24 thanks for documenting this!

**Pull request merged**